config/init_db.sql

```diff
@@ -1,9 +1,15 @@
 # This file is executed immediately after mysql_install_db,
 # to initialize a fresh data directory.
 
-##########################################
+###############################################################################
+# WARNING: This sql is *NOT* safe for production use,
+#          as it contains default users and passwords
+#          with more privileges than necessary.
```

Comment:
As far as I know, the privileges given are necessary. Is there an example you see that's unnecessary?

The main thing that we intended for people to change but didn't document well enough is the passwords, especially for the users that allow remote connections.

Reply:
I've removed comment about privileges, and added comment about rotating them as necessary.